tests/roles/dataplane_adoption/tasks/nova_ffu.yaml

```diff
@@ -0,0 +1,196 @@
+- name: get database service environment variables
+  ansible.builtin.include_role:
+    name: mariadb_copy
+    tasks_from: env_vars.yaml
+
+# FIXME(bogdando): catch up with ongoing osdpd ansible variables support work in progress
+# and make sure the Nova compute service is deployed with nova_ffu_workarounds.yaml task
+# from the beginning of EDPM adoption (by the original osdpd/openstack).
+- name: configure pre-FFU workarounds for Nova compute EDPM services to update its version records
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc apply -f - <<EOF
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: nova-compute-workarounds
+      namespace: openstack
+    data:
+      19-nova-compute-cell1-workarounds.conf: |
+        [workarounds]
+        disable_compute_service_check_for_ffu=true
+    EOF
+
+    oc apply -f - <<EOF
+    apiVersion: dataplane.openstack.org/v1beta1
+    kind: OpenStackDataPlaneService
+    metadata:
+      name: nova-compute-workarounds
+      namespace: openstack
+    spec:
+      label: nova.compute.workarounds
+      configMaps:
+        - nova-compute-workarounds
+      playbook: osp.edpm.nova
+    ---
+    apiVersion: dataplane.openstack.org/v1beta1
+    kind: OpenStackDataPlaneDeployment
+    metadata:
+      name: openstack-nova-compute-workarounds
+      namespace: openstack
+    spec:
+      nodeSets:
+        - openstack
+      servicesOverride:
+        - nova-compute-workarounds
+    EOF
+
+    oc patch osdpns/openstack -n openstack --type=json --patch \
```

Comment:
I didn't know that patching osdpns may be not required. I'll re-test this, with using a serviceOverride for it only